openstack_lease_it/lease_it/backend/TestConnection.py

```diff
@@ -138,7 +149,10 @@ def _domains(self):
 
         :return: dict()
         """
-        response = cache.get('domains')
+        if GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
```

Comment:
Remplacer par
if not GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
response = cache.get('instances')